C4G.Unity/Assets/C4G/Editor/Scripts/CodeGenerator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Text;
+
+namespace C4G.Editor
+{
+    public static class CodeGenerator
+    {
+        public static string GenerateDTOClass(ParsedSheet parsedSheet)
+        {
+            var sb = new StringBuilder();
+
+            sb.AppendLine($"public sealed class {parsedSheet.Name}");
```

Comment:
[Minor] It's better to leave it public partial to allow to extend outside from the codeGen.